app/models/paper.rb

```diff
@@ -1,16 +1,16 @@
 class Paper < ApplicationRecord
   attr_accessor :locator_id, :locator_type
+
   acts_as_taggable
   acts_as_taggable_on :biases, :methodologies
 
   has_and_belongs_to_many :authors
   has_many :lists, through: :references
   has_many :references, dependent: :destroy
   has_many :links, dependent: :destroy
-  has_many :api_import_responses
+  has_many :api_import_responses, dependent: :destroy
```

Comment:
looks like you chose for dependent destroy, probably the wise thing to do for now until we have a good reason to save them.